.github/pull_request_template.md

```diff
@@ -1,10 +1,10 @@
-**Please delete this line before submitting**, _Pull request TITLE should look like this_: `[Resource] -> [Resource Section In Docs]`
+`[ApexCharts.js] -> [Javascript Chart Libraries]`
```

Comment:
Please revert this file

Reply:
Hi, I have reverted this file. I'm really sorry for that.

Reply:
No problems, Thanks for the pull request! 🎉